css/style.css

```diff
@@ -145,3 +156,154 @@ button:active:enabled {
   justify-content: space-around;
   padding: 20px 0;
 }
+
+.works-wrapper {
+  display: grid;
+  grid-template: repeat(4, 1fr) / 1fr;
+  background-color: var(--background);
+  padding: 114px 24px;
+  gap: 114px;
+}
+
+.work-card {
+  width: 100%;
+  height: auto;
+  background-color: var(--white);
+  border: 1px solid var(--border);
+  border-radius: 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 16px;
+  gap: 12px;
+}
+
+.work-card_picture {
+  height: auto;
+  width: 100%;
+  border-radius: 8px;
+}
+
+.work-card_picture > img {
+  width: 100%;
+}
+
+.work-card_title {
+  width: 100%;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  margin: 0;
+}
+
+.work-card_button {
+  margin: 12px 0;
+}
+
+.work-card_specs {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.work-card_specs > .specs_role,
+.work-card_specs > .specs_year {
+  color: #7a869a;
+}
+
+.work-card_specs .specs_client {
+  color: #344563;
+  text-transform: uppercase;
+}
+
+.dot {
+  background-color: var(--gray);
+  width: 8px;
+  height: 8px;
+  border-radius: 4px;
+  margin: 0 12px;
+}
+
+.work-card_description {
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.work-card_techs {
+  display: flex;
+  width: 100%;
+  column-gap: 8px;
+}
+
+.tag {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  background: var(--tag-background);
+  border-radius: 8px;
+  font-size: 12px;
+  line-height: 16px;
+  color: var(--tag-color);
+  letter-spacing: 0.03em;
+}
+
+.about-myself {
+  border-top-right-radius: 70px;
+  display: flex;
+  flex-direction: column;
+}
+
+.about-myself > .social_wrapper {
+  margin: 3px 0;
+}
+
+.about-myself h4 {
+  font-size: 20px;
+  font-weight: 500;
+  color: var(--black);
+  margin: 0 0 0 12px;
+}
```

Comment:
- [ ] Great job with your design :clap: Kindly make sure you add the icon beside the `language` text 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/205040140-6d805f42-9d5e-4deb-9892-0aa5f85bae32.png)  |  ![image](https://user-images.githubusercontent.com/74814780/205040294-dcf72d49-3c38-4b76-844c-23d93af6e6e8.png)

Reply:
Thanks for showing me. The comment bubbles in Figma had hidden the arrow.